### DISCUSSION TITLE
Edge function returned invalid JWT in local development

### DISCUSSION THREAD

**Discussion by hallee9000** [supabase/supabase#6758] [General]
I'm trying edge function in local development. I always got an error saying 'Invalid JWT' when requesting the endpoint. Here are the specific results I got when trying in different env or keys.

| |  local | production |
|  ----  | ----  | ----  |
| anon key | Invalid JWT | success |
| service_role | Invalid JWT | success |
| example key in doc* | success | Invalid JWT |

\* example key in doc is the authorization key in edge function doc:
<img width="927" alt="image" src="https://user-images.githubusercontent.com/12868055/167593274-eb7fd374-a85a-4f26-9b12-6a50deb72d07.png">

**Comment by ijaspreetsinghh** [comment]
I think JWT for local is different from that provided in supabase API. so for local requests you need to use JWT provided locally, while creating a new function, its commented code is below the function body.

![image](https://user-images.githubusercontent.com/44422128/167594713-a7d46b8e-3ec2-48df-a6f6-3a13641f945c.png)

**Reply by hallee9000** [reply]
I guess it is.
